src/flow/src/adapter/flownode_impl.rs

```diff
@@ -92,6 +95,35 @@ impl Flownode for FlowWorkerManager {
                     .map_err(to_meta_err)?;
                 Ok(Default::default())
             }
+            Some(flow_request::Body::Flush(FlushFlow {
+                flow_id: Some(flow_id),
+            })) => {
+                // TODO(discord9): impl individual flush
+                debug!("Starting to flush flow_id={:?}", flow_id);
+                // wait 100ms so that the source data is sended to our source
+                // so case like we run too quickly before input data have chance to be sended
+                // TODO(discord9): found better way to impl this flush
+                tokio::time::sleep(Duration::from_millis(100)).await;
```

Comment:
Does it mean that the flush request arrives at flownode later than the mirror request?

Reply:
> Does it mean that the flush request arrives at flownode later than the mirror request?

refactor to use a rwlock instead which `read` when handle_inserts and `write` when `flush_flow`